cms/templates/cms/home_page.html

```diff
@@ -80,16 +80,16 @@ <h2>
         <div class="col-md-12 ">
           <section class="sub-banner-section">
             <div class="sub-banner-text">
-            <h3>What is the MicroMasters Credential?</h3>
+              <h3>What is the MicroMasters<span>&#174;</span> Program Credential?</h3>
             <div class="youtubevideowrap">
               <div class="video-container">
                 <iframe width="560" height="315" src="https://www.youtube.com/embed/7lVvOSJpFGY" frameborder="0" allowfullscreen></iframe>
               </div>
             </div>
-              <p>MicroMasters is a professional and academic credential for online learners from anywhere in the world. Learners who pass an integrated set of MITx graduate-level courses on edX.org, and one or more proctored exams, will earn a MicroMasters credential from MITx, and can then apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
-              <p>MicroMasters courses offer the same learning and challenge as MIT courses.</p>
+              <p>A MicroMasters program credential is a professional and academic credential for online learners from anywhere in the world. Learners who pass an integrated set of MITx graduate-level courses on edX.org, and one or more proctored exams, will earn a MicroMasters program credential from MITx, and can then apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
```

Comment:
```suggestion
              <p>A MicroMasters program credential is a professional and academic credential for online learners from anywhere in the world. Learners who pass an integrated set of MIT’s graduate-level courses on edX.org, and one or more proctored exams, will earn a MicroMasters program credential from MITx, and can then apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
```

Feel free to break this into multiple lines while you're in here, if you like.